scripts/configureApplication.js

```diff
@@ -4,56 +4,15 @@ const fs = require('fs');
 const path = require('path');
 const childProcess = require('child_process');
 const chalk = require('chalk');
+const { config } = require('dotenv');
+
+config();
+
+const { SENTRY_DSN } = process.env;
 
 const log = message => console.log(`➤ ${message}`); // eslint-disable-line no-console
 const error = message => console.log(chalk.red(`➤ ${message}`)); // eslint-disable-line no-console
 
-// TODO: fetch .env variables from Vault (or maybe just store the .env file itself in the Vault - this is painful)
```

Comment:
Removed because this was dead code, right?

Reply:
Yes, I think it is better to be able to actually use the `.env` than waiting around for something that we have not been able to achieve in 1.5 years 🙈